### DISCUSSION TITLE
Saving favorite quotes

### DISCUSSION THREAD

**Discussion by tsironis** [monkeytypegame/monkeytype#2812] [Ideas] [answered]
It would be nice to be able to see all quotes that a user has saved in the past in a single view or page.

I can see that there is an option to practice using only saved quotes (screenshot below) but I think they are different usecases.

![Screenshot 2022-07-08 at 8 55 29 PM](https://user-images.githubusercontent.com/782448/178045201-da4a500a-e79c-437a-893c-5ce05638fbe9.png)

**Comment by mak448a** [comment, marked as answer]
It's already there. It's the heart by the flag and the star after you finish typing a quote.

**Reply by tsironis** [reply]
I'm sorry I didn't phrase my suggestion correctly. I am aware but I can't seem to find a way to see all the quotes in a single page or list. 

My original comment has been edited accordingly.

**Reply by mak448a** [reply]
You can see all of the quotes that you have saved in the past by going to the quote search and pressing the Favorites Only button.
